deployer/app-deployer/pkg/api/requestHandler.go

```diff
@@ -0,0 +1,90 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/config"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/deployer/app"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/utils"
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	APP_DEPLOYER_PATH = "/deploy/application"
+	HOST              = "0.0.0.0"
+	POST              = "POST"
+	GET               = "GET"
+	CONTENT_TYPE      = "application/json"
+	APPLICATION_NAME  = "app-deployer"
+	STATUS            = "up"
+)
+
+var (
+	err          error
```

Comment:
err it's used only once in this file, so I don't see the need for a "static" declaration, instead of declaring it where it is used
also, I would be a bit concerned about this approach of declaring err at package level, as it represents a static status of the whole package and can be changed unexpectedly in other functions. I see you used the same pattern in other files, is it coming from some recommendations that I missed?

Reply:
As the `var err error` is a package-level variable declaration, it is applicable to all files in the package, including services.go where `err` is also applied. 
Declaring a variable like `var err error` seems to be a common practice in Go, i have seen it implicated in many examples. It basically declares a variable and sets an initial zero value, which is `nil`. 
Since we are calling multiple functions which will ultimately initialize `err` with `nil` it might make sense not to declare `err` every time we do so, this approach can help reduce code duplication and make the code more readable.
In addition, since in Go it is a customary pattern to follow every initialization of `err` by `err != nil` (or alike) to check if the value is `nil` to determine if an error occurred and launch a dedicated response, all changes to the `err` variable are tracked consistently.
All this said, it is not considered a best practice nor worst practice, but just a common practice between Go developers. If this is still a cause for concern, i can use the basic `err :=...` format instead, its not a problem.